stdlib/public/stubs/Stubs.cpp

```diff
@@ -60,6 +63,9 @@ static long double swift_strtold_l(const char *nptr,
 #define strtod_l swift_strtod_l
 #define strtof_l swift_strtof_l
 #define strtold_l swift_strtold_l
+#elif defined (_MSC_VER)
+extern int *__argc;
```

Comment:
There already was a `defined(_MSC_VER)` condition in this `#if` chain.  I think it would be better to not mix these declarations into the locale block, and instead add it to the `#if` chain where you declare `_NSGetArgv`.